content/launch-keyboard.md

```diff
@@ -0,0 +1,68 @@
+hidden: false
+section: software
+---
+
+## The Launch and Accessories
+
+The following items are included with your Launch:
+
+![Accessories](/files/launch-keyboard/launch-accessories.webp)
+
+## QMK firmware
+
+The Launch keyboard uses our version [QMK firmware](https://github.com/system76/qmk_firmware) and it is built using our changes that are not in upstream [QMK firmware](https://github.com/qmk/qmk_firmware). You can read more about QMK [here](https://github.com/system76/qmk_firmware).
+
+## Layers
+
+The core idea of "Layers" is that one key can be used in different setups for example on our laptops holding the <kbd>Fn</kbd> then pressing the <kbd>F5</kbd> will lower the volume while pressing <kbd>F5</kbd> on it's own will perform a different action like refreshing a web page if you are in a web browser like <u>Firefox</u>. You can read more about layers [here](https://beta.docs.qmk.fm/using-qmk/software-features/feature_layers
+).
+
+## Firmware Updates
+
+Firmware updates for the Launch are handled using [fwupd](https://fwupd.org/) and our <u>Firmware Manager</u> can be used to apply the update. When you get a notification about a firmware update you will need to unlock the device using Fn+Esc (note that this will be the default keys even if you remapped them in the Keyboard Configurator) before you can apply the update. When the keyboard is unlocked you will see the following keys light up:
+
+- U N L O C K E D
+
+Once the keyboard is in this mode you can apply the firmware update, when the update is complete you will need to unplug and re-plug it in to exit this mode.
+
+## Keyboard Configurator
+
+You can download the `AppImage` for Ubuntu and other OSes and use the <u>Pop!\_Shop</u> for Pop!\_OS. Use this [link](https://github.com/pop-os/keyboard-configurator/releases
+) to download the application.
+
+You can also download it from the Arch Linux AUR. The main version is built from latest releases which is recommended for most users which you can find [here](https://aur.archlinux.org/packages/keyboard-configurator/
+) and the git version is built from the latest commits to the project (this version will have the very latest fixes and features) which can be found [here](https://aur.archlinux.org/packages/keyboard-configurator-git/
+).
+
+## Remapping Keys
+
+<video width="900" height="900" controls>
+  <source src="/files/launch-keyboard/remapping-function-keys.webm" type="video/mp4">
+</video>
+
+The keys are defined in the firmware of your keyboard and you can remap the keys using the <u>Keyboard Configurator</u>. After opening the <u>Keyboard Configurator</u>, you can then select the key you would like to configure, and select an item from the available items below the keyboard showing in the application. I am attaching a video showing the remapping process. Please note the "Layer 1" and "Layer 2" at the top of the configurator. Layer 1 is what the keys will do normally, then Layer 2 shows what the keys do when "Fn" is held down.
```

Comment:
`you can remap the keys using the <u>Keyboard Configurator</u>` is an independent clause, so add a comma before "and" in the first sentence.

> you can then select

The word `then` is not needed here. It may make sense if this was the beginning of the sentence, but it's not.

> select the key you would like to configure, and select an item

I don't think there should be a comma before this "and". You're listing two things, the second section isn't functioning as an independent clause.

> I am attaching a video showing the remapping process.

You should not be referring to yourself in the first person. Something like `See the attached video showing the remapping process.` would be an alternative.

> Layer 1 is what the keys will do normally, then Layer 2 shows what the keys do when "Fn" is held down.

There is no time component to these buttons, so I would recommend using "and" or another word instead of "then."